docs/quick_start/starting.rst

```diff
@@ -38,3 +40,20 @@ In both cases the user will be prompted to enter a password. It is possible, but
    The *start*, *status* and *help* commands do not require authentication.
 
 Following a successful authentication attempt a time based token is issued that allows the user to run further commands, for a limited time, without the need to authenticate again.
+
+Authentication File
+-------------------
+
+The prompting for username and password when using the *fledge* script can be bypassed if an authentication file is created. This is a file that should be created in a directory called *.fledge* in the user's home directory.
+
+The file created should be called *auth* and contains two lines, the first line is the username to use to login, the second line is the password for that user.
+
+The *auth* file will only be read if the permissions on that file are set such that only the owner can read the file.
```

Comment:
We can include this as a note or highlight it, as it is a necessary condition that must be met.